CMakeLists.txt

```diff
@@ -780,7 +780,7 @@ if (USE_BREAKPAD)
     endif()
 endif()
 
-option(PREFER_EXTERNAL_LIBS "Tries to use system libs where possible." ON)
+option(PREFER_EXTERNAL_LIBS "Tries to use system libs where possible. Only works for Linux" ON)
```

Comment:
"Only works for Linux" is not really true. For example on Mac CMake can find things with the Homebrew package manager; on Windows from the MSYS package manager.